### PR TITLE
Use istio-proxy 1.13.2

### PR DESCRIPTION
**Pull request by yitzchake**
1. Update helm charts to use Use Curiefense with istio-proxy 1.13.2

### CODE REVIEW

File: istio-helm/charts/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.descriptor      linguist-generated=true
+*.descriptor      -diff -merge
+*.descriptor_set  linguist-generated=true
+*.descriptor_set  -diff -merge
+*.pb.html linguist-generated=true
+*.pb.go linguist-generated=true
+*.gen.go linguist-generated=true
+*.gen.yaml linguist-generated=true
+*_pb2.py linguist-generated=true
```
Comment (xavier-rbz) [inline istio-helm/charts/.gitattributes:9]: this file should not be there

File: curiefense-helm/curiefense/charts/curiefense-admin/templates/curietasker-deployment.yaml [hunk 2]
```diff
@@ -1,43 +0,0 @@
-apiVersion: apps/v1
```
Comment (xavier-rbz) [inline curiefense-helm/curiefense/charts/curiefense-admin/templates/curietasker-deployment.yaml:1]: Removing curietasker and updating istio a completely independent, unrelated things. Let's have 1 PR for each, plaese.

**Review by xavier-rbz — CHANGES_REQUESTED** [review]

File: curiefense-helm/curiefense/charts/curiefense-admin/templates/curietasker-deployment.yaml (hunk 2 above)
Comment (yitzchake) [reply curiefense-helm/curiefense/charts/curiefense-admin/templates/curietasker-deployment.yaml]: Restored

File: istio-helm/charts/.gitattributes (hunk 1 above)
Comment (yitzchake) [reply istio-helm/charts/.gitattributes]: Removed

**Pull request closed without merging**